EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc

```diff
@@ -151,7 +151,7 @@ void GEMRawToDigiModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
       // best to skip these events since FED is most likely corrupt
       edm::LogWarning("GEMRawToDigiModule")
           << "FED trailer: fail check? " << failTrailerCheck << " fail match? " << failTrailerMatch;
-      continue;
+      //  continue;
```

Comment:
@jshlee Attention: is this something that should be in the 11_3_X?

Reply:
it's best to keep the continue.
or we can make it python configurable to skip bad FEDs

Reply:
@cecilecaillol @jshlee 
please clarify what will be done here.
It looks like this edit is the only reason for the reco signature.
Thank you.

Reply:
I have kept the continue in the new PR. I hope this is ok for you now.